payu_config/archive_scripts/standardise_mom6_filenames.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/bash
+# Copyright 2024 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: Apache-2.0.
+#
+# Standardise file naming for MOM6 output files from access-om3.
```

Comment:
I think this comment could be more explicit. i.e. replaces `_YYYY` with `.YYYY`

Reply:
```suggestion
# Standardise file naming for MOM6 output files in access-om3 by removing the underscore before the four-digit year, i.e., replacing '_YYYY' with 'YYYY'
```